.gitlab-ci.yml

```
@@ -26,23 +28,41 @@ variables:
     --time=00:10:00 
     --account=SHELLARD-SL3-GPU 
     --partition=ampere
+  FCOMPARE_FLAGS: >
+    --abs_tol 1e-10
+    --rel_tol 1e-10
+    --abort_if_not_all_found
+  LOCKED_FILENAME: >
+    file.lock
+
 
 csd3-a100:
   stage: test
   tags:
     - csd3
   script:
+    - module purge
+    - module use ${MODULEPATHS_A100}
+    - module load ${MODULES_A100}
     - if [[ -d ../amrex ]]; then
     -   cd ../amrex
     -   git fetch --depth 1
     -   git reset --hard origin/development
     - else
     -   git clone --depth 1 https://github.com/AMReX-Codes/amrex.git ../amrex
     - fi
+    - AMREX_HOME="$PWD"
+    - cd ${AMREX_HOME}/Tools/Plotfile
```

I think this might not work if we end up in the `else` branch as the current directory will be wrong for the final `cd`.

We should probably set `AMREX_HOME` earlier and then use that consistently so maybe something like the following (I haven't tested it):

```suggestion
    - export AMREX_HOME=$(realpath ${HOME}/${CI_PROJECT_DIR}/../amrex)
    - if [[ -d ${AMREX_HOME} ]]; then
    -   cd ${AMREX_HOME}
    -   git fetch --depth 1
    -   git reset --hard origin/development
    -   cd -
    - else
    -   git clone --depth 1 https://github.com/AMReX-Codes/amrex.git ${AMREX_HOME}
    - fi
    - cd ${AMREX_HOME}/Tools/Plotfile
```